src/sentry/mediators/mediator.py

```diff
@@ -161,6 +162,10 @@ def record_analytics(self):
         # used to record data to Amplitude
         pass
 
+    def post_install(self):
```

Comment:
I think we should rename this, mediators technically don't need to be associated with the sentry apps at all. they just happen to used by them. I think `post_call` is less sentry app specific

Reply:
@MeredithAnya I renamed it to `post_commit` since there are actually other things that happen after `call` but before things are committed (analytics for one).